src/ucp/wireup/select.c

```diff
@@ -1120,8 +1165,38 @@ ucp_wireup_add_rma_bw_lanes(ucp_wireup_select_ctx_t *select_ctx)
             continue;
         }
 
-        ucp_wireup_add_bw_lanes(select_ctx, &bw_info,
-                                context->mem_type_access_tls[mem_type]);
+        ucp_wireup_bw_info_set_iface_flags(context, &bw_info, 0);
+        status = ucp_wireup_add_bw_lanes(select_ctx, &bw_info,
+                                         context->mem_type_access_tls[mem_type]);
+        ucp_wireup_bw_info_unset_iface_flags(context, &bw_info, 0);
```

Comment:
can we use just set/reset flag? `unset_flags` looks excessive

Reply:
done